tests/dummy/lib/ember-blog/addon/routes/post.js

```diff
@@ -13,6 +13,20 @@ export default Ember.Route.extend({
   actions: {
     goToChineseVersion() {
       this.transitionTo({ queryParams: { lang: 'Chinese' } });
+    },
+    transitionToHome() {
+      this.transitionToExternal('home')
+        .then(() => {
+          var postController = this.controllerFor(this.routeName);
+          postController.set('transitionedToExternal', true);
```

Comment:
Ah, good point. Can we at least use normal navigation (e.g., `click`) to get back to the page? I'm not really a fan of multiple `visit` calls in a single test as that is not how an actual user will navigate the site.